machine_translation/README.md

```diff
@@ -0,0 +1,28 @@
+# Encoder-Decoder with search for machine translation.
+In this demo, encoder-decoder architecture with attention mechanism is used for
+machine translation. The attention mechanism is implemented according to
+[1]. The training data used is WMT15 Czech to English corpus, which you have
+to download, preprocess and put to your `datadir` in the config file. Note
+that, you can use `prepare_data.py` script to download and apply all the
```

Comment:
We should take it out of `__main__` if it's here, and refer people to the
README instead. No point in having the same text two places as it will
eventually become out of sync.

On Wed, Nov 11, 2015 at 5:30 PM, Eder Santana <notifications@github.com>
wrote:

> In machine_translation/README.md
> <https://github.com/mila-udem/blocks-examples/pull/56#discussion_r44596629>
> :
>
> > @@ -0,0 +1,28 @@
> > +# Encoder-Decoder with search for machine translation.
> > +In this demo, encoder-decoder architecture with attention mechanism is used for
> > +machine translation. The attention mechanism is implemented according to
> > +[1]. The training data used is WMT15 Czech to English corpus, which you have
> > +to download, preprocess and put to your `datadir` in the config file. Note
> > +that, you can use `prepare_data.py` script to download and apply all the
>
> this was a copy paste from *main* let me fix it in there as well
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/blocks-examples/pull/56/files#r44596629>.
>